src/js/components/covid19/DataSourcesAndMethodology.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const DataSourcesAndMethodology = () => (
+    <div className="heading__container information-body">
+        <div className="information-top" />
+        <div className="dsm-section">
+            <h2 className="dsm-topHeading">Data Sources &amp; Methodology</h2>
```

Comment:
we use [BEM](http://getbem.com/naming/) naming conventios so this should likely be


```jsx
<div className="dsm__top-heading"
```

We also use kebob casing rather than camel casing (`ke-bob-case`) and snake casing for distinguishing between a block and element (`dsm__top-heading`) (`snake_case_ing_things_like_this`)